src/main/MQ2ImGuiConsole.cpp

```diff
@@ -64,6 +64,15 @@ static bool s_consoleVisible = false;
 static bool s_consoleVisibleOnStartup = false;
 static bool s_resetConsolePosition = false;
 static bool s_setFocus = false;
+static bool s_isConsoleLocked = false;
+
+static bool s_isTitleBarShown = false;
```

Comment:
My Problem Is I cant not figure out how to Move these into Member for instance Into Line 938, ImGuiConsole 
int m_alphaConsoleNormal;
And access it in 1544 	if (ImGui::SliderInt("##Alpha Normal", &m_alphaConsoleNormal, 0, 100))
	{
		WritePrivateProfileInt("MQConsole", "AlphaNormalOnStartup", m_alphaConsoleNormal, mq::internal_paths::MQini);
	}